pkgs/development/libraries/qt-5/modules/qtbase.nix

```
@@ -29,6 +29,9 @@
 
 let
   debugSymbols = debug || developerBuild;
+  qtPlatform = plat: with plat;
```

```suggestion
  qtPlatformCross = plat: with plat;
```
This should make it clearer that we don't need none cross entries here.